packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -307,8 +308,22 @@ export class PluginsPage {
 	 * @param {string} expectedPluginName Name of the plugin to validate.
 	 */
 	async validateConfirmationPagePostInstall( expectedPluginName?: string ): Promise< void > {
-		await this.page.getByRole( 'heading', { name: "You're all set", exact: false } ).waitFor();
-
+		await this.page
+			.getByRole( 'heading', { name: "Congrats on your site's new superpowers!" } )
+			.click();
+
+		// Check for expiration text
+		await this.page.locator( selectors.installedPluginCard ).getByText( 'expire' ).waitFor();
+
+		// Check for the correct footer cards
+		await this.page.locator( selectors.installedfooterCards ).getByText( 'Keep growing' ).waitFor();
+		await this.page.locator( selectors.installedfooterCards ).getByText( 'Learn More' ).waitFor();
```

Comment:
Looks good! Should we preemptively remove this check due to [this PR](https://github.com/Automattic/wp-calypso/pull/74796)?

Reply:
Yes, I'll do it, thanks!